packages/framer-motion/src/value/use-on-change.ts

```diff
@@ -1,18 +1,18 @@
-import { useEffect } from "react"
+import { useIsomorphicLayoutEffect } from "../three-entry"
```

Comment:
@mattgperry importing this from `../three-entry` seems like it would create a circular import or something wouldn't it?